components/http_foundation/trusting_proxies.rst

```diff
@@ -11,9 +11,9 @@ Trusting Proxies
 
 If you find yourself behind some sort of proxy - like a load balancer - then
 certain header information may be sent to you using special ``X-Forwarded-*``
-headers. For example, the ``Host`` HTTP header is usually used to return
-the requested host. But when you're behind a proxy, the true host may be
-stored in a ``X-Forwarded-Host`` header.
+headers or the ``Forwarded`` header. For example, the ``Host`` HTTP header is
+usually used to return the requested host. But when you're behind a proxy,
+the true host may be stored in a ``X-Forwarded-Host`` header.
```

Comment:
Maybe replace "true host" with "actual host" here? (my dev. mind strongly binds true to a boolean value)